scoreboard-colors.scss

```diff
@@ -28,6 +28,59 @@ $color-night: #0F171D;
 $color-warning: #FBCC3B;
 // -- END: COLOR PALATTE
 
+// V6 UPDATED COLOR PALATE
+// -- START: COLOR PALATTE
+$color-gray-025: #fcfcfd;
+$color-gray-050: #f9fafb;
+$color-gray-100: #f5f7fa;
+$color-gray-150: #eef1f6;
+$color-gray-200: #e6e9f2;
+$color-gray-250: #dde0eb;
+$color-gray-300: #d5d9d5;
+$color-gray-400: #c8cbd9;
+$color-gray-450: #a4a8ba;
+$color-gray-500: #818599;
+$color-gray-550: #6b6f80;
+$color-gray-600: #545766;
+$color-gray-700: #494d5c;
+$color-gray-750: #3c3f4d;
+$color-gray-800: #303440;
+$color-gray-900: #242733;
+$color-gray-950: #171923;
+$color-gray-950-4: rgba(23, 25, 35, 0.04);
+$color-gray-950-8: rgba(23, 25, 35, 0.08);
+$color-gray-950-14: rgba(23, 25, 35, 0.14);
+$color-orange-0250: #fffdfa;
+$color-orange-050: #fff8ec;
+$color-orange-100: #ffedd3;
+$color-orange-200: #ffdaa6;
+$color-orange-300: #ffbf75;
+$color-orange-400: #ffa64d;
+$color-orange-500: #ff9124;
+$color-orange-600: #f57a00;
+$color-orange-700: #e26003;
+$color-orange-750: #bf480d;
+$color-orange-800: #943b0f;
+$color-orange-900: #612405;
+$color-orange-950: #3a1503;
+$color-ai-violet-025: #fbfaff;
+$color-ai-violet-050: #eeebff;
+$color-ai-violet-100: #d8cdfe;
+$color-ai-violet-200: #c4a8f5;
+$color-ai-violet-300: #9f72ee;
```

Comment:
@scottoreilly @derekstory @nathanbowser for my own understanding, do the designs really use all of these various shades of gray, orange, etc? Or are these just baselines that go into all code automatically generated by Figma?

Reply:
It seems like they're all intended to be added as new colors but maybe I'm misunderstanding something? Here's the design:

<img width="1354" alt="Screenshot 2024-01-03 at 9 38 19 AM" src="https://github.com/SpiderStrategies/scoreboard-colors/assets/3791116/5d85cecb-ddcd-461a-a84b-f7d1b2572b1d">

Reply:
Ah hah. Thanks for the screenshot!

Reply:
Yessss, @derekstory Thanks for the screenshot! Scott showed me that yesterday and everything made sense.